networking/ciao-cnci-agent/network.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	"gopkg.in/yaml.v2"
 
+	"github.com/coreos/go-iptables/iptables"
```

Comment:
it may make sense to actually put the code in firewall.go and call it from here... as it may be useful in other places.

Take a look at following places where the palce holders for this code already exists
https://github.com/01org/ciao/blob/master/networking/libsnnet/firewall.go#L281
https://github.com/01org/ciao/blob/master/networking/libsnnet/firewall.go#L341

All the code you wrote here should move into firewall.go and modify the existing placeholders